code/app/src/main/java/com/example/the_tarlords/data/users/User.java

```diff
@@ -29,13 +31,16 @@ public class User implements Profile {
     private CollectionReference usersRef = MainActivity.db.collection("Users");
 
     private String fCMToken;
+    private Boolean isAdmin;
+
+
 
     /**
      * Mandatory empty constructor for firestore functionality
      */
     public User() {
     }
-
+    //TODO : automatically sets isAdmin to false in constructor, should we have a constructor that allows us to choose, or we changing in firebase directly?
```

Comment:
We will be getting it from firebase directly, but firebase uses the empty constructor to create objects so setting admin to false in this constructor shouldn't matter.